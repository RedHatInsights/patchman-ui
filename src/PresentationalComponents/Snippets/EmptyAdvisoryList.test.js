import EmptyAdvisoryList from './EmptyAdvisoryList';
import toJson from 'enzyme-to-json';
import { shallow } from 'enzyme';

describe('EmptyAdvisoryList', () => {
    it('Should match the snapshot', () => {
        const wrapper = shallow(<EmptyAdvisoryList />);
        expect(toJson(wrapper)).toMatchSnapshot();
    });
});
