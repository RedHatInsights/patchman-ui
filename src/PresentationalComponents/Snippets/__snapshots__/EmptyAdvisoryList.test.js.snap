// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EmptyAdvisoryList Should match the snapshot 1`] = `
<EmptyState
  variant="full"
>
  <EmptyStateIcon
    icon={[Function]}
  />
  <Title
    headingLevel="h5"
    size="lg"
  >
    No matching Advisories found
  </Title>
</EmptyState>
`;
